### PR TITLE
Support markdown report in Gradle plugin

### PR DESCRIPTION
**Pull request by VitalyVPinchuk**
For #4967

### CODE REVIEW

**Comment by VitalyVPinchuk** [comment]
Why is it failing on Windows 11?

**Comment by BraisGabin** [comment]
Windows is flaky. Don't worry about it.

File: README.md [hunk 1]
```diff
@@ -76,6 +76,7 @@ tasks.withType<Detekt>().configureEach {
         xml.required.set(true) // checkstyle like format mainly for integrations like Jenkins
         txt.required.set(true) // similar to the console output, contains issue signature to manually edit baseline files
         sarif.required.set(true) // standardized SARIF format (https://sarifweb.azurewebsites.net/) to support integrations with Github Code Scanning
+        md.required.set(true) // simple Markdown format
```
Comment (cortinico) [inline README.md:79]: Don't forget to update the docs on the website as well 👍

**Review by cortinico — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (VitalyVPinchuk) [reply README.md:79]: Hey @cortinico
I need some help
`docs/` folder is untracked by git, should changed files be added?

Comment (BraisGabin) [reply README.md:79]: All of them are in `website`.

File: website/docs/introduction/reporting.md [hunk 2]
```diff
@@ -37,7 +37,12 @@ XML is a machine-readable format that can be integrated with CI tools. It is com
 [SARIF](https://sarifweb.azurewebsites.net/) is a standard format for the output of 
 static analysis tools. It is a JSON format with a defined 
 [schema](https://docs.oasis-open.org/sarif/sarif/v2.0/csprd02/schemas/). It is currently supported
-by Github Code Scanning and we expect more consuming tools will be adopt this format in the future.
+by GitHub Code Scanning, and we expect more consuming tools will adopt this format in the future.
+
+### MD
```
Comment (cortinico) [inline website/docs/introduction/reporting.md:42]: That's a good start, but we'll have to update also:
1. The snippets in this page
2. The `website/docs/intro.mdx` page
3. The `docs/gettingstarted/gradle.mdx`

File: README.md (hunk 1 above)
Comment (VitalyVPinchuk) [reply README.md:79]: fixed

**Review by BraisGabin — APPROVED** [review]

**Comment by BraisGabin** [comment]
Thanks for the PR!

**Pull request merged**